build/common.sh

```diff
@@ -393,8 +393,9 @@ function kube::build::docker_build() {
   local -r image=$1
   local -r context_dir=$2
   local -r pull="${3:-true}"
-  local -r build_args=$4
-  local -ra build_cmd=("${DOCKER[@]}" build -t "${image}" "--pull=${pull}" "${build_args}" "${context_dir}")
+  # shellcheck disable=SC2206
```

Comment:
Disable lint should have a comment about why we disabled it.
We could instead choose here to read to an array splitting on space explicitly

Reply:
Alright, I went into the shell check docs to get a workaround. It might be a little messy because I had to make the variable local first and then set the value then make it readonly. If you have a suggestion for a better way, let me know

Reply:
a comment would be OK, or we can do something like:
https://github.com/kubernetes/kubernetes/blob/23af91b2938fae7c4f556160ec473d62903700e6/hack/lib/util.sh#L800

(sorry, I should have suggested something more concrete last night!)